In [1]:
# importowanie stats z modułu scipy - korzystasz z funkcji statystycznych dostępnych w bibliotece SciPy
from scipy import stats

# Zadanie 2 - Test dla wartości oczekiwanej pojedynczej prób

W celu zmniejszenia zużycia paliwa Komisja Doradcza Przewozów Morskich zleciła
kompaniom dalekomorskim zredukowanie prędkości statków do 17 węzłów.

Jedna z kompanii, której statki pływają z reguły z prędkością od 18,5 do 19,8 węzłów przy dziennym
średnim zużyciu paliwa w wysokości 127,7 t, sprawdziła rejestr rejsów i odnalazła 5 takich
rejsów, dla których średnia prędkość wynosiła zaledwie 17 węzłów. 

Dzienne zużycie paliwa w czasie tych rejsów wynosiło: 

101,1; 105,7; 102,6; 113,4; 98,1 t. 

Zakładając, że dzienne zużycie paliwa ma rozkład normalny, czy dane te potwierdzają słuszność Komisji, że obniżenie
prędkości do 17 węzłów zmniejszy zużycie paliwa poniżej 127,7, przy poziomie istotności
0,01?


## 1. Sformułowanie hipotez


H0: μ = 127,7 (przeciętne zuzycie paliwa nie zmieni sie)\
H1: μ < 127,7 (przeciętne zuzycie paliwa zmniejszy sie)


## 2. Wybór testu statystycznego
- Test parametryczny - wartość przeciętna
- Pojedyncza próba
- Skala ilorazowa

-> Test T

## 3. Poziom istotności  
α=0,01


## 4. Obliczenie statystyki testowej i p-value oraz podjęcie decyzji weryfikacyjnej

In [5]:
# Wagi netto próbki
zuzycie_paliwa = [101.1, 105.7, 102.6, 113.4, 98.1]

# Wartość oczekiwana
mu = 127.7

# Przeprowadź test t-studenta
statystyka_T, p_wartosc = stats.ttest_1samp(zuzycie_paliwa, mu, alternative="less")

# Wyświetl wyniki
print("Statystyka testowa:", statystyka_T)
print("P-wartość:", p_wartosc)

# Sprawdź istotność statystyczną na poziomie alfa = 0.05
alfa = 0.01
if p_wartosc < alfa:
    print("Odrzucamy hipotezę zerową - średnie zużycie paliwa jest mniejsze niż 127.7 litra.")
else:
    print("Nie ma podstaw do odrzucenia hipotezy zerowej - średnie zużycie paliwa jest większe lub rowne 127.7 litra.")

Statystyka testowa: -9.010634513141104
P-wartość: 0.00042000583374171395
Odrzucamy hipotezę zerową - średnie zużycie paliwa jest mniejsze niż 127.7 litra.


Z otrzymanego wydruku możemy odczytać m.in.  wartość statystyki testowej (t = -9.010634513141104) oraz  p-wartość = 0.00042000583374171395. \
Ponieważ: p<α, odrzucamy hipoteze zerowa. Srednie zuzycie paliwa jest znacznie mniejsze niz 127.7 litra.


## 5. Wnioski:
Srednie zuzycie paliwa jest znacznie mniejsze niz 127.7 litra

